### ISSUE TITLE
DirectoryNotFoundException: Could not find a part of the path - Exporter 2.17.0-pre

### ISSUE THREAD

**Issue by ROBYER1** [needle-tools/needle-engine-support#44]
Getting these errors in a project migrated from 2.16~ Exporrter to 2.17.0-pre

```
DirectoryNotFoundException: Could not find a part of the path "C:\Unity_Projects\ProjName\Temp\Needle\Export\Debug\MyGltf.debug.json".
—
System.IO.DirectoryNotFoundException: Could not find a part of the path "Temp/Needle/Export/Debug/MyGltf.debug.json".
```

```
DirectoryNotFoundException: Could not find a part of the path "C:\Unity_Projects\ProjName\Library\Basic\cache~\2f03f3a162fd0d24389f92f0a3ef46a1.hash".
—
System.IO.DirectoryNotFoundException: Could not find a part of the path "Library/Basic/cache~/2f03f3a162fd0d24389f92f0a3ef46a1.hash".

```

**Comment by marwie** [comment]
Thanks for reporting, it's fixed locally and will be out in a bit later today

**Comment by ROBYER1** [comment]
> Thanks for reporting, it's fixed locally and will be out in a bit later today

Does it also fix UI buttons exporting as white squares once fixed? Was about to log that bug too with this version

**Comment by ROBYER1** [comment]
Fixed in 2.17.1-pre
